### ISSUE TITLE
Error handling and mongoose validation errors

### ISSUE THREAD

**Issue by simonschllng** [KunalKapadia/express-mongoose-es6-rest-api#192]
During development it is useful to see the full error reported by mongoose. E.g. you forget to assign a required parameter.

At the moment I only get `ValidationError: MyModel validation failed` and a useless stack trace (all methods are part of the error handling module).

Mongoose adds details in the `err.errors` property.

What would be the best way to get these error details through the error handling process (in `APIErrors.js`)?

**Comment by duard** [comment]
Same problem  here

**Comment by mxmzb** [comment]
This is a large eyesore for me, does anyone have an idea how to handle this?

**Comment by okonon** [comment]
I agree with everyone here. I tried to add correct stacktrace but ran out of time and was assigned to diff project.

**Comment by arikogan** [comment]
+1

**Comment by okonon** [comment]
@KunalKapadia do you have any suggestions how to proceed?
